test/unit/vl-map-search.test.html

```diff
@@ -205,6 +322,207 @@
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched met: een coördinaat met enkel een eindhaakje', () => {
+        const searchInput = '104719.27; 192387.25)';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched met: een coördinaat met spaties ervoor', () => {
+        const searchInput = '     104719.27; 192387.25)';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched met: een coördinaat zonder spatie na het scheidingsteken', () => {
+        const searchInput = '     104719.27;192387.25)';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched met: een coördinaat met meerdere spaties na het scheidingsteken', () => {
+        const searchInput = '104719.27;     192387.25)';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched met: een coördinaat met 3 cijfers na de komma', () => {
+        const searchInput = '104719.271, 192387.252';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched met: een coördinaat met 5 cijfers na de komma', () => {
+        const searchInput = '104719.27123, 192387.25456';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
+
+        expect(result[0]).to.be.equal(searchInput);
+        expect(result.groups.x).to.be.equal('104719.27');
+        expect(result.groups.y).to.be.equal('192387.25');
+      });
+
+      test('Matched niet met: een vrije tekst', () => {
+        expect('Gent'.match(LAMBERT_COORDINAAT_REGEX)).to.be.null;
+      });
+
+      test('Matched niet met: een lege string', () => {
+        expect(''.match(LAMBERT_COORDINAAT_REGEX)).to.be.null;
+      });
+
+      test('Matched niet met: een spatie', () => {
+        expect(' '.match(LAMBERT_COORDINAAT_REGEX)).to.be.null;
+      });
+    });
```

Comment:
Kunnen deze testen niet op functioneel lambert-coordinaat niveau getest worden? Als je voor al de mogelijke functionele varianten een correct Lambert coördinaat kan aanmaken, maakt de achterliggende REGEX op zich niet uit. Het is zelfs zo dat deze testen puur de REGEX testen, maar de lambert-coordinaat class achterliggend misschien toch een andere REGEX gebruikt.

Reply:
Ah, zelfde comment als ik hierboven ...